src/components/common/ColonyActions/ActionDetailsPage/ActionDetailsPageFeed/ActionDetailsPageFeed.tsx

```diff
@@ -17,14 +17,17 @@ const ActionDetailsPageFeed = ({
   actionData,
 }: // networkEvents,
 ActionsPageFeedProps) => {
-  const events = ACTIONS_EVENTS[actionData.type];
+  const events =
+    JSON.parse(actionData.individualEvents as string) ||
```

Comment:
I would leave a comment here explaining why this is safe (i.e. how you know that actionData.individualEvents won't be undefined and cause the JSON.parse to error)

Reply:
Sure, but I'm not going to bother with this since we're going to change this logic anyway.